HAB5/hab5.tex

```diff
@@ -0,0 +1,422 @@
+Understanding of basic circuits and connections/interfacing is required for both.
+An knowledge of heat transfer basics is recommended in order to do basic thermal analysis. A team of 2-3 mechanicals would be required for the project itself. Additional talent can be used to probe the question of camera stabilization and other long-term niceties.
+The HAB structure leverages HAB4 designs so the brunt of mechanical design will focus on camera mounting and enclosures.
+University Physics II (with optics) is recommended for imaging systems testing and calibration tasks.
+
+\subsection{Funding Requirements}
+\label{funding}
+% Payload:  ~$200 (hard cap at $300)
+% 2x raspberry pi 3 = $100 @ $40-50 each
+% 1x pi camera v2 = $28
+% 1x pi camera noir = $25
+% An arduino FPGA is a possible but unlikely inclusion, with a price tag of $60. The present budget also leaves room for the physical hardware of the payload including camera mounts and possible printing expenses. A filter setup, a la TetraPi, is deemed to be prohibitively expensive.
+
+% 	1x vis filter (reddish) $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB650-40
+% 	1x nir filter = $100 https://www.thorlabs.com/thorproduct.cfm?partnumber=FB880-70
+
+% 2x rpi $100 refly ones
+% 2x pi cam $60
+% hab avionics (from last year) $150
+% helium $120
+% balloon $100
+The budget described in \autoref{budget} assume the worst case where no material is reused from HAB4, and uses HAB4 and WUAP to baseline costs.
+From this baseline cost estimate, the components and materials for HAB5 and PLANTHAB module should not exceed \$600.
+
+\begin{table}[ht]
+  \caption{Estimated cost}
+  \centering
+  \begin{tabular}{@{}llc@{}}
+    \label{budget}
+    \textbf{Item} & \textbf{Cost} & \textbf{Reused?} \\
+    \midrule
+    Balloon & \$100 & No \\
+    Helium & \$120 & No\\
+    Avionics, sensors \& batteries & \$150 & Yes\\
+    (2x) Raspberry Pi 3 & \$100 & Yes\\
+    Pi Camera V2 & \$30 & Yes \\
+    Pi Camera NoIR & \$30 & No \\
+    Foam \& structural materials & \$50 & Yes \\
+    \midrule
+    \textit{Total} & \$580 & \\
+  \end{tabular}
+\end{table}
+
+This estimate is based off of the inclusion of a prior HAB board for baseline functionality, an existing APRS module and some functional WUAP hardware.
+Significant leeway was left specifically because the exact implementation topology (for processing) and level of pre-fabrication of components scales inversely with availability of time dedicated by advanced electrical engineering students.
+
+\subsection{Faculty Support}
+The project laid forth doesn't assume that faculty support is guaranteed or that any particular faculty members will be forthcoming.
+The project is undertaken in part hoping to build these connections without any significant one-way reliance, along questions discussed in \autoref{Essential questions}.
```

Comment:
Recompile to get the references to generate. Sometimes you have to hit build a couple times for the references to show up. 

Short answer: Rerun it. Default settings should be doing it right.

Long answer: We are using BibLaTeX to make bibliographies. When compiling, the build process (behind the scenes) executes the build process `latex bibtex latex latex`. So it compiles normally, then compiles the bib, then compiles normal latex twice. Don't know why but this is what it needs to do. The Atom LaTeX package already does this by default. https://stackoverflow.com/questions/2461905/compiling-latex-bib-source